output_http.go

```diff
@@ -184,6 +184,12 @@ func (o *HTTPOutput) Read(data []byte) (int, error) {
 
 func (o *HTTPOutput) sendRequest(client *HTTPClient, request []byte) {
 	meta := payloadMeta(request)
+
+	if len(meta) < 2 {
```

Comment:
Meta is constructed manually, and always should be (type, uuid, time), do you know a repeatable case when meta gets corrupted (because it never should) ?